### PR TITLE
Update Documentation related to Customise Feature for v1.3

### LINKED ISSUE #134: Week 11: Update documentation related to Customise Features
Update documentation related to Customise Order Feature

### PR DESCRIPTION
**Pull request by PangKuangWei**
1. Fix customise order feature diagram
2. Update user guide - make it more clear
3. Update developer guide
4. Update testing.md docs file paths

### CODE REVIEW

File: docs/diagrams/CustomiseOrderSequenceDiagram.puml [hunk 1]
```diff
@@ -1,21 +1,18 @@
 @startuml
 !include style.puml
+scale 1550*500
 
 box Logic LOGIC_COLOR_T1
 participant ":LogicManager" as LogicManager LOGIC_COLOR
 participant ":SoConnectParser" as SoConnectParser LOGIC_COLOR
 participant ":CustomiseCommandParser" as CustomiseCommandParser LOGIC_COLOR
 participant ":CustomiseOrderCommandParser" as CustomiseOrderCommandParser LOGIC_COLOR
 participant "u:CustomiseOrderCommand" as CustomiseOrderCommand LOGIC_COLOR
```
Comment (Eugene-Ong-W-X) [inline docs/diagrams/CustomiseOrderSequenceDiagram.puml]: The instance of CustomiseOrderCommand is named `u` but when returned, it becomes `c`?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -239,16 +241,18 @@ Example:
...
-Customise order of information shown in all contacts shown.
+Customise the order of information shown in all contacts shown.
 
-Format: `customise order [p/] [e/] [a/] [t/]`
+Format: `customise order [t/] [p/] [e/] [a/]`
 
-* Attributes not given will be ordered last in default order. (tag > phone > email > address)
+* Name of contact will always be at the top of each contact and cannot have its order changed.
+* Information that can have its order changed: Tags, Phone Number, Email, Address.
+* Information that are not specified will be ordered last in default order. (Tags > Phone Number > Email > Address)
```
Comment (Eugene-Ong-W-X) [inline docs/UserGuide.md]:
```suggestion
* Information that are not specified will be ordered last and follow the default order. (Tags > Phone Number > Email > Address)
```

**Review by Eugene-Ong-W-X — COMMENTED** [review]
Great work

File: docs/diagrams/CustomiseOrderSequenceDiagram.puml (hunk 1 above)
Comment (PangKuangWei) [reply docs/diagrams/CustomiseOrderSequenceDiagram.puml]: Ok, nice catch.

File: docs/UserGuide.md (hunk 2 above)
Comment (PangKuangWei) [reply docs/UserGuide.md]: Ok, I shall change it.

**Pull request merged**